### PR TITLE
View tech filters comfortably on mobile

### PR DESCRIPTION
**Pull request by hnipps**
Added a button to expand and contract the list of tech filters, so this feature takes up less screen space. Mainly designed for mobile use.

### CODE REVIEW

File: src/application/components/technologies/technology-sidebar/technology-sidebar.js [hunk 1]
```diff
@@ -44,7 +45,6 @@ class TechnologySidebar extends Component {
   };
 
   addTechnology = event => {
-    event.preventDefault();
```
Comment (nkxy) [inline src/application/components/technologies/technology-sidebar/technology-sidebar.js]: Is this no longer needed? I think I saw this in the button.js file too.

Comment (hnipps) [reply src/application/components/technologies/technology-sidebar/technology-sidebar.js]: Yeah it's not needed anymore. I'm looking into why.

**Comment by nkxy** [comment]
I'm noticing some odd behaviour with what order it adds the technologies after a certain point. I added letters in alphabetical order, and it placed them in order up until k, where it starts randomly shifting already placed technologies. Is this supposed to happen? 
screen shot 2018-09-10 at 4 35 41 pm
screen shot 2018-09-10 at 4 35 57 pm

**Comment by hnipps** [comment]
No that's not supposed to happen. I thought I'd gotten rid of that issue. It should sort the tags like this: priority tech goes first, in the same order specified in the array of priority tech, and then everything else comes after in the order they were received from the db.

**Comment by hnipps** [comment]
It doesn't happen in Safari, but does in Chrome. Haven't checked Firefox.

**Comment by nkxy** [comment]
screen shot 2018-09-10 at 4 53 53 pm
screen shot 2018-09-10 at 4 54 01 pm
The sorting doesn't seem to work, even on safari for me. I added two agents to "l"

**Comment by hnipps** [comment]
It's not sorted by number of agents with a skill, there's a hardcoded array of priority skills (see technology-sidebar.js ~line 123). Those skills are first in the list, everything else goes afterwards.

**Comment by hnipps** [comment]
The issue in Chrome should be fixed by my latest commit.

I think the issue was that I was accessing the technologies array during sorting. Now I'm creating a map of tech and priority before sorting, and using that priority in the sort method. So there's no need to access the technologies array during sorting.

**Pull request merged**